docs/usage.md

```diff
@@ -347,6 +349,28 @@ Specify to run Bismark with the `--slam` flag to run bismark in [SLAM-seq mode](
 
 Specify to run Bismark with the `--local` flag to allow soft-clipping of reads. This should only be used with care in certain single-cell applications or PBAT libraries, which may produce chimeric read pairs. (See [Wu et al.](https://doi.org/10.1093/bioinformatics/btz125) (doesn't work with `--aligner bwameth`)
 
+### `--bismark_align_cpu_per_multicore`
+
+The pipeline makes use of the `--multicore` option for Bismark align. When using this option,
+Bismark uses a large number of CPUs for every `--multicore` specified. The pipeline
+calculates the number of `--multicore` based on the resources available to the task.
+It divides the available CPUs by 3, or by 5 if `--single_cell`, `--zymo` or `--non_directional`
```

Comment:
Does this mean 3 if either of the two first and 5 if `--non_directional` is specified? It's a bit ambiguous the way it's currently written

Reply:
3 unless any of those three are specified..

Reply:
But yes I agree, this was not clear 👍 Hopefully improved now..

Reply:
Yes, much better.